src/stavenote.ts

```diff
@@ -164,11 +164,41 @@ export class StaveNote extends StemmableNote {
       });
     }
 
-    const voices = notesList.length;
+    let voices = 0;
+    let noteU = undefined;
+    let noteM = undefined;
+    let noteL = undefined;
+    const draw = [false, false, false];
+
+    for (let i = 0; i < notesList.length; i++) {
+      draw[i] = notesList[i].note.render_options.draw == false ? false : true;
```

Comment:
FYI, you can remove this loop:

`const draw = notesList.map((n) => !(n.note.render_options.draw))`